spark/v3.0/spark/src/main/java/org/apache/iceberg/spark/actions/BaseMigrateTableSparkAction.java

```diff
@@ -221,4 +230,12 @@ private void restoreSourceTable() {
           e);
     }
   }
+
+  private void dropBackupTable() {
+    try {
+      destCatalog().dropTable(backupIdent);
+    } catch (Exception e) {
+      LOG.error("Cannot drop the backup table {} after migrate is completed.", backupIdent, e);
```

Comment:
Nit: grammar error here (should be- after + noun).  How about,  'Failed to drop the backup table {} after the migration'

Reply:
I am not sure I understand the grammar error here, but I think it matches the style of the rest of the error messages in the file. But I changed `after migrate ` to `after migration`.

Reply:
Can you change the rest of the Spark files?   "after" should be used before a noun, and "migrate" is a verb.

Reply:
I think the latest commit addresses the grammar issue correctly.  I used an online grammar checker to make sure the latest version is grammatically correct. Please let me know if a change is needed there. Thanks.

Reply:
Sounds good, it's a bit long but its fine.  Ref for the grammar comment (missed putting it earlier):  https://dictionary.cambridge.org/us/grammar/british-grammar/after-afterwards